src/edu/stanford/dlss/wowza/SulWowza.java

```diff
@@ -149,11 +149,28 @@ String getStacksUrl(IApplicationInstance appInstance)
         }
     }
 
+    // the user's actual IP address will be the first entry in a comma-separated list
+    // of IP addresses in the "x-forwarded-for" header (as there might be proxies between
+    // the user and wowza).
+    // returns an empty string if it can't parse out an IP address.
+    String getUserIp(java.util.Map<String,String> httpReqHeaders)
```

Comment:
Is there any reason why you didn't just import java.util.Map, so you could leave out the fully qualified class name here?